doc/TBG_macros.cfg

```diff
@@ -186,7 +186,8 @@ TBG_checkpoints="--checkpoints 1000"
 
 # Restart the simulation from checkpoints created using TBG_checkpoints
 TBG_restart="--restart --restart-step 1000"
-
+# To restart in a new run directory point to the old run where to start from
+#   --restart-directory /path/to/simOutput
```

Comment:
So I don't forget to update it when I have the time.

(Poor mans multi tasking.)